rst.rst

```diff
@@ -23,7 +27,9 @@ Call the ``function_name`` function.
 Code blocks
 ===========
 
-Look at this code::
+Look at this code:
+
+.. code-block:: python
```

Comment:
I understand wanting to show how to do the syntax coloring, but let's also keep the simpler form.

If you have many code blocks in the same language, you can do this:
```
.. highlight:: python

Code sample 1::

    def foo():
        return 1

Code sample 2::

    def bar():
        return 2
```
Both code blocks will be syntax-colored as Python.  We can show all of these options.

Reply:
Seems to me like showing all three syntaxes would be an awful lot, I wanted to show the best and most directly translated version on each side; I'll see if there's a good link I could put in the notes for more variations

Reply:
I added an example of the "simple" version back in

Reply:
I can live without the `.. highlight::` syntax in the cheat sheet.  Showing the two you have is right for this sheet.